var/spack/repos/builtin/packages/gr-osmosdr/package.py

```diff
@@ -0,0 +1,35 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+class GrOsmosdr(CMakePackage):
+    """While primarily being developed for the OsmoSDR hardware, this
+    block as well supports:
+
+    FunCube Dongle through libgnuradio-fcd
+    FUNcube Dongle Pro+ through gr-fcdproplus
+    sysmocom OsmoSDR Devices through libosmosdr
+    Nuand LLC bladeRF through libbladeRF library
+    Great Scott Gadgets HackRF through libhackrf
+    Ettus USRP Devices through Ettus UHD library
+    Fairwaves UmTRX through Fairwaves' fork of Ettus' UHD library
+    RFSPACE SDR-IQ, SDR-IP, NetSDR (incl. X2 option)
+    RTL2832U based DVB-T dongles through librtlsdr
+    RTL-TCP spectrum server (see librtlsdr project)
+    MSi2500 based DVB-T dongles through libmirisdr
+    SDRplay RSP through SDRplay API library
+    AirSpy R820t dongles through libairspy
+    gnuradio .cfile input through libgnuradio-blocks"""
```

Comment:
The newlines will be stripped in `spack info gr-osmosdr`, might want to add commas or semi-colons